src/main/java/lotto/domain/CustomLottoGenerator.java

```diff
@@ -0,0 +1,23 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+public class CustomLottoGenerator implements LottoGenerator {
+
+    private final List<Integer> customLotto;
+
+    public CustomLottoGenerator(List<Integer> customLotto) {
+        this.customLotto = customLotto;
+    }
+
+    @Override
+    public Lotto generateLotto() {
+        Set<LottoNumber> lottoNumbers = customLotto.stream()
+                .map(lottoNumber -> LottoNumber.of(lottoNumber)).collect(Collectors.toSet());
```

Comment:
개인적으로는 stream은 가독성을 위해 한줄마다 줄 바꿈을 해주고 있어요! 
```suggestion
        Set<LottoNumber> lottoNumbers = customLotto.stream()
                .map(lottoNumber -> LottoNumber.of(lottoNumber))
                .collect(Collectors.toSet());
```